src/components/themed/TransactionListTop.tsx

```diff
@@ -458,25 +426,25 @@ const getStyles = cacheStyles((theme: Theme) => ({
     fontSize: theme.rem(2)
   },
   settingsIcon: {
-    alignSelf: 'center'
+    alignSelf: 'flex-end'
   },
   // Send/Receive Buttons
   buttonsContainer: {
+    flex: 1,
     marginTop: theme.rem(1),
-    flexDirection: 'row'
+    flexDirection: 'row',
+    justifyContent: 'space-evenly'
```

Comment:
I'm guessing this like changed the button layout from left justified. Given that the buttons have no border, it looks a bit weird to me and I'm on the fence about the change. If this was approved by design, then ok, otherwise I'd like to revert back.

Reply:
I don't see the relation between borders and left justification? Left justification seems less friendly to internationals, and the spacing always looked strange to me in this section.

I just did a fine tune session with Madison and we tweaked it a bit to ignore the card's built in spacing. She also likes this centered style better than the legacy layout.